sdk/keyvault/Azure.Security.KeyVault.Certificates/tests/LightweightPkcs8DecoderTests.cs

```diff
@@ -0,0 +1,157 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Security.Cryptography;
+using System.Text;
+using Azure.Core;
+using NUnit.Framework;
+
+namespace Azure.Security.KeyVault.Certificates.Tests
+{
+    public class LightweightPkcs8DecoderTests
+    {
+        [Test]
+        public void VerifyECDecoderPrime256v1Imported() =>
+            // The following signature was generated in Key Vault using the certificate key corresponding to EcPrime256v1PrivateKeyImported,
+            // thus proving that Key Vault-generated OID 1.2.840.10045.1.1 (prime-field) is equivalent to OID 1.3.132.0.10 (secp256k1).
```

Comment:
No, it only proves that you're seeing an explicit encoding for that particular curve, not that all explicit encodings are that particular curve.

Reply:
Agreed. This will change when I update the manual curve processing.